src/views/admin/PersonDetailsView.vue

```diff
@@ -272,7 +295,22 @@
     return !!rolle && rolle.Rollenart === RollenArt.Lern;
   }
 
-  const validationSchema: TypedSchema = toTypedSchema(
+  // Check if the button to change the Klasse should be active or not. Activate only if there is 1 selected Zuordnung and if it is of type LERN.
+  const canChangeKlasse: ComputedRef<boolean> = computed(() => {
+    const hasOneSelectedZuordnung: boolean = selectedZuordnungen.value.length === 1;
```

Comment:
Gibt es einen Fall, in dem mehr als eine Klasse zugeordnet sein kann?

Reply:
Ist die Bedingung dann notwendig? Weil dann gibt es ja immer nur eine Zuordnung

Reply:
Hier wird eher geprüft wie viel Zuordnungen selektiert wurden.. so: 
![image](https://github.com/user-attachments/assets/20e752b9-080f-416d-9e03-3a2e5356dce4)

Wenn mehr als eine Zuordnungen selektiert wurde dann wird der Button "Klasse ändern" deaktiviert.

Reply:
Verstanden. Danke